### PR TITLE
Remove unnecessary test adapters

### PR DESCRIPTION
**Pull request by rli**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-ultimate/tst/software/aws/toolkits/jetbrains/services/redshift/RedshiftUtilsTest.kt [hunk 1]
```diff
@@ -13,12 +14,13 @@ import software.aws.toolkits.core.utils.RuleUtils
 import software.aws.toolkits.jetbrains.core.MockResourceCacheExtension
 import software.aws.toolkits.jetbrains.core.region.getDefaultRegion
 import software.aws.toolkits.jetbrains.services.sts.StsResources
-import software.aws.toolkits.jetbrains.testFramework.ProjectExtension
 
 class RedshiftUtilsTest {
-    @JvmField
-    @RegisterExtension
-    val projectRule = ProjectExtension()
+    companion object {
```
Comment (manodnyab) [inline jetbrains-ultimate/tst/software/aws/toolkits/jetbrains/services/redshift/RedshiftUtilsTest.kt:19]: Why is this moved to a companion object?

Comment (rli) [reply jetbrains-ultimate/tst/software/aws/toolkits/jetbrains/services/redshift/RedshiftUtilsTest.kt:19]: it's the equivalent of a class rule. the extension only runs before/after the entire suite

**Review by manodnyab — APPROVED** [review]
